app/mailers/thing_mailer.rb

```diff
@@ -1,9 +1,15 @@
 class ThingMailer < ActionMailer::Base
   default from: 'adoptadrain@codeforsanfrancisco.org'
 
+  def adopted_confirmation(thing)
+    @thing = thing
+    @user = thing.user
+    mail(to: @user.email, subject: ['Thank you for Adopting a Drain in San Francico'])
```

Comment:
We can definitely split that part out to a separate issue. Maybe @hackajesse or @jasonlally can comment about languages?